validate_version.sh

```diff
@@ -10,5 +10,9 @@ TAGS="$(git tag -l)"
 
 
 # check if current version already has a tag which means the publish will fail
-[[ $TAGS =~ (^|[[:space:]])$CV_WITH_V($|[[:space:]]) ]] && echo "a release tag with version $CV_WITH_V already exists" && exit 1
+[[ $TAGS =~ (^|[[:space:]])$CV_WITH_V($|[[:space:]]) ]] && \
+echo "a release tag with version $CV_WITH_V already exists - updating version and rerunning CI" && \
+npm version patch -m "Bumping patch version after CI version conflict" && git push && \
```

Comment:
@MickaelAlliel not sure that's a good idea.

You don't know the new one is free, and having that stuff committed is probably a bad idea.

Reply:
Usually we bump version manually in a PR when we want to release. This is juuust in case we forget or push a readme fix. This is what we do with all the SDKs as well